src/web_app/js/call.js

```diff
@@ -419,23 +419,45 @@ Call.prototype.maybeCreatePcClient_ = function() {
   if (this.pcClient_) {
     return;
```

Comment:
It hits this check 3 times when another user joins the call (via startSignaling_ which is called on in getTurnServicersPromise_ and onRemoteHangup. Also via onRecvSignalingChannelMessage_ which is hooked up to the this.channel_.onmessage event) but not before. Also when a user (B) leaves and rejoins, the user (A) left in the call hits this 2 check times while the user (B) rejoins.

What comment would be appropriate here? E.g.
```
// Resolve promise if pcClient already exists.
```